UI/js-src/lsmb/ReconciliationLine.js

```diff
@@ -0,0 +1,32 @@
+define("lsmb/ReconciliationLine",
+       ["dojo/_base/declare",
+        "dojo/on",
+        "dijit/registry",
+        "dojo/dom-class",
+        "dijit/form/CheckBox"],
+       function(declare, on, registry, domClass, CheckBox) {
+           return declare("lsmb/ReconciliationLine", [CheckBox], {
+               publish: function(targetValue) {
```

Comment:
Misnomer. Renamed to _display. That handles the display of cleared, mismatch or outstanding, based on a message received from a ToggleButton